artifacts/definitions/Windows/Timeline/Prefetch.yaml

```diff
@@ -29,19 +28,17 @@ parameters:
       type: timestamp
     - name: binaryRegex
       description: "Regex of executable name."
-      type: regex
```

Comment:
Lets put those back in

Reply:
done - i need to remember to add these in on my test instance.